tests/unit/src/test/resources/semanticdb/example/Definitions.scala

```diff
@@ -1,15 +1,18 @@
 package example
 
+import io.circe.derivation.deriveDecoder/*io.circe.derivation.package.deriveDecoder(+2).*//*io.circe.derivation.package.deriveDecoder(+1).*//*io.circe.derivation.package.deriveDecoder().*/
+import io.circe.derivation.deriveEncoder/*io.circe.derivation.package.deriveEncoder(+2).*//*io.circe.derivation.package.deriveEncoder(+1).*//*io.circe.derivation.package.deriveEncoder().*/
+
 class Definitions/*example.Definitions#*/ {
   Predef/*scala.Predef.*/.any2stringadd/*scala.Predef.any2stringadd().*/(1)
-  List/*scala.collection.immutable.List.*/[
+  List/*scala.package.List.*/[
     java.util.Map/*java.util.Map#*/.Entry/*java.util.Map#Entry#*/[
       java.lang.Integer/*java.lang.Integer#*/,
       java.lang.Double/*java.lang.Double#*/
     ]
   ](
-    xs/*scala.collection.immutable.List.apply().(xs)*/ = null
+    elems/*scala.collection.IterableFactory#apply().(elems)*/ = null
   )
-  println/*scala.Predef.println(+1).*/(MacroAnnotation/*example.MacroAnnotation.*/.decodeMacroAnnotation/*example.MacroAnnotation.decodeMacroAnnotation.*/)
-  println/*scala.Predef.println(+1).*/(MacroAnnotation/*example.MacroAnnotation.*/.encodeMacroAnnotation/*example.MacroAnnotation.encodeMacroAnnotation.*/)
+  println/*scala.Predef.println(+1).*/(deriveDecoder/*io.circe.derivation.package.deriveDecoder().*//*local0*/[MacroAnnotation/*example.MacroAnnotation#*/])
```

Comment:
Is `/*local0*/` expect actually, is this coming from an implicit parameter?

Reply:
Yea this is another one I had a tricky time with since the way circe derivation was working here in 2.13 was way different so I did what I _thought_ would test the same thing, but I'm actually a bit unsure.